packages/origin-ui-core/src/components/exchange/order-book-order.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { Product, Bid, Ask } from '@energyweb/exchange-core';
+
+export class OrderBookOrderDTO {
+    price: number;
+
+    volume: string;
+
+    product: Product;
+
+    userId: string;
+
+    public static fromOrder(order: Bid | Ask, userId?: string): OrderBookOrderDTO {
```

Comment:
You don't need that method. Remove it, this will also allow you to remove @energyweb/exchange-core dependency.